core/src/main/java/com/graphhopper/reader/dem/USGSProvider.java

```diff
@@ -41,21 +41,56 @@ boolean isOutsideSupportedArea(double lat, double lon) {
 
     @Override
     double getMinLatForTile(double lat) {
-        return 37.75;
+        return (double) Math.floor(lat * 4) / 4;
```

Comment:
You can remove these casts, they're redundant because `Math.floor()` returns a `double`.